src/runtime-rs/crates/hypervisor/Cargo.toml

```diff
@@ -68,3 +71,7 @@ hypervisor = { path = ".", features = ["cloud-hypervisor"] }
 test-utils = { path = "../../../libs/test-utils" }
 
 serial_test = "2.0.0"
+
+
+[build-dependencies]
+ttrpc-codegen = "0.4.2"
```

Comment:
Please don't drop the terminating `\n` .

Reply:
Fixed